src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -365,39 +371,44 @@
       {:token                                     token-by-symbol
        :send-amount-in-crypto                     amount-in-crypto
        :valid-input?                              valid-input?
-       :token-not-supported-in-receiver-networks? token-not-supported-in-receiver-networks?
+       :token-not-supported-in-receiver-networks? unsupported-token-in-receiver?
```

Comment:
reanamed the var
`token-not-supported-in-receiver-networks?` to  `unsupported-token-in-receiver?`

The first one is too large and it made oru code formatting uglier :(